common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -24,6 +24,9 @@
 from ...pages.lms.tab_nav import TabNavPage
 from ...pages.lms.teams import TeamsPage, MyTeamsPage, BrowseTopicsPage, BrowseTeamsPage, CreateOrEditTeamPage, TeamPage
 
+#TODO: merge this import into the above line properly once merged with Peter's delete changes
+#do not merge to master as-is
+from ...pages.lms.teams import EditMembershipPage
```

Comment:
This was just to avoid merge conflicts, once the delete changes and this live on the same branch I'll fix it up. See https://github.com/edx/edx-platform/blob/peter-fogg/delete-teams/common/test/acceptance/tests/lms/test_teams.py#L25